content/java/concepts/array-list/terms/indexOf/indexOf.md

```diff
@@ -0,0 +1,92 @@
+---
+Title: '.indexOf()'
+Description: 'If ArrayList contains given element returns the index of the first occurrence of it , else -1'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Arrays'
+  - 'Data Types'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.indexOf()`** method returns the index of the first occurrence of the specified element in [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list), or -1 if it does not contain the element
+
+## Syntax
+
+```pseudo
+arrayListInstance.indexOf(element);
+```
+
+Signature of the method is `indexOf(Object element)` where `element` is the element whose lowest index is to be returned.
+
+it does not throw any exception if the element is not present in the [`ArrayList`](https://www.codecademy.com/resources/docs/java/array-list) but returns -1.
+
+but `.equals()` method may throw a NullPointerException if the element is null.
+
+## Example
+
```

Comment:
```suggestion

The following example features two calls to `.indexOf()` on `animals`. The first will print the index of the first occurrence of the element with the string `"Tiger"`. The second will return -1 because `"Elephant"` does not exist in `animals`:

```